src/utils/tokens.ts

```diff
@@ -410,7 +441,25 @@ const tokens = {
     const outputs: IDataOutput[] = [];
     const tokens = [authorityMeltInput.token];
     const depositPercent = storage.getTokenDepositPercentage();
-    const withdrawAmount = this.getWithdrawAmount(amount, depositPercent);
+    let withdrawAmount = this.getWithdrawAmount(amount, depositPercent);
+    // The deposit amount will be the quantity of data strings in the array
+    // multiplied by the fee or 0 if there are no data outputs
+    let depositAmount = data != null ? this.getDataScriptOutputFee() * data.length : 0;
```

Comment:
Why are you using `!=` null here? It's safe to use `!==` as we are defaulting it to `null`, right?

Reply:
Changed